### ISSUE TITLE
Typescript

### ISSUE THREAD

**Issue by f-solecki** [Tosunami/react-date-countdown-timer#26]
Is there any plans for adding TS for the library?
Or maybe someone did it before and can share it with me?

**Comment by cengizhanbasak** [comment]
Good idea, I am planning to work on this. PRs are accepted as well if you want to contribute.
